net/etherwake-nfqueue/Makefile

```diff
@@ -8,13 +8,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=etherwake-nfqueue
-PKG_RELEASE:=2
+PKG_RELEASE:=3
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://github.com/mister-benjamin/etherwake-nfqueue.git
-PKG_SOURCE_DATE:=2019-09-28
-PKG_SOURCE_VERSION:=f71c269b58585e93575fa3e9fcc1793806fb3080
-PKG_MIRROR_HASH:=4960dc592abc4ca06504c92ca09fc736c678353df0dcc32d4081e17b137a9164
+PKG_SOURCE_DATE:=2023-06-20
+PKG_SOURCE_VERSION:=857c8714c4aec2b54ae70655c6d0e13f3c70ed0a
+PKG_MIRROR_HASH:=skip
```

Comment:
Could you generate and fill in the hash?

Reply:
Oh sorry I did that already but I think I forgot to commit it. Will do